src/components/CalendarMonthGrid.jsx

```diff
@@ -91,25 +98,32 @@ function getMonths(initialMonth, numberOfMonths, withoutTransitionMonths) {
   return months;
 }
 
-export default class CalendarMonthGrid extends React.Component {
+class CalendarMonthGrid extends React.Component {
   constructor(props) {
     super(props);
     const withoutTransitionMonths = props.orientation === VERTICAL_SCROLLABLE;
     this.state = {
       months: getMonths(props.initialMonth, props.numberOfMonths, withoutTransitionMonths),
     };
 
+    this.calendarMonthHeights = new Array(props.numberOfMonths);
```

Comment:
eesh, `new Array` creates a sparse array, which is terribad for performance. does this need to start out as more than `[]`?

Reply:
I was doing this because I didn't want to assume the order that the `CalendarMonth` heights would come back in. As such, I'm creating the array with the known length and then setting `this.calendarMonthHeights[i] = ...` as the heights come back. Is there a better way to do this? I suppose I could use an object instead but that seems... silly

Reply:
you don't have to set the initial length, JS will grow it for you as you set `i` - but also you could do `Array.from({ length: props.numberOfMonths })` if you want a dense array of `undefined` values.

Reply:
oh, i did not know that you could do 
```
const foo = [];
foo[3] = 'bar';
```
and that would work. 

I think it just goes against all my computer science degree sensibilities. :P

Reply:
If the values of this array are all positive ints, you might want to use a typed array.

Reply:
a what now

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays

But you'd probably need to have a regular array fallback for older browsers. Probably outside the scope of this PR, but possibly worth looking into for the future.